nav2_smac_planner/src/node_hybrid.cpp

```diff
@@ -337,6 +337,11 @@ float HybridMotionTable::getAngleFromBin(const unsigned int & bin_idx)
   return bin_idx * bin_size;
 }
 
+unsigned int getNumOfBins()
+{
+  return num_angle_quantization;
```

Comment:
You could just use `num_angle_quantization` for the one instance this method is used? That seems in style with the rest of it. The table is treated like a global struct to grab info (for better or worse)